src/sonic-yang-models/tests/yang_model_tests/tests_config/bgp.json

```diff
@@ -0,0 +1,1305 @@
+{
+    "VRF_VALID": {
```

Comment:
Why VRF Test is part of this PR? just curious.

Reply:
Will remove it, when I raised this PR, VRF table was not present in the community, so, added VRF table & it's UTs in the PR.